torch/csrc/distributed/c10d/cuda/utils.cpp

```diff
@@ -1,3 +1,5 @@
+#include <cuda_runtime.h>
```

Comment:
This is needed in order for the `CUDART_VERSION` macro to be defined. Without this include the conditional below would always fail and `CUDART_SUPPORTS_MULTICAST` would always be undefined.

Reply:
Good catch!

Reply:
@kwen2501 Do you know why unit tests did not fail when symmetric memory mc was broken?
Maybe the multicast tests are skipped if hw does not supported, It would be great to have at least one test running on a known config that never gets skips to catch those kind of bugs.